packages/shared/package.json

```diff
@@ -44,7 +44,8 @@
     "jss": "^10.10.0",
     "jss-plugin-sort-css-media-queries": "^1.0.1-beta.0",
     "jss-preset-default": "^10.10.0",
-    "puppeteer": "~21.6.1",
+    "pixelmatch": "^5.3.0",
```

Comment:
I think we don't need pixelmatch anymore because all VRT tests are executed by Playwright?

Reply:
You are right, this must have been a merge issue. I will remove it.